packages/react-console/src/components/VncConsole/VncConsole.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+
+import { EmptyState, EmptyStateBody, EmptyStateIcon, Spinner } from '@patternfly/react-core';
+
+import * as NovncLog from 'novnc-core/lib/util/logging';
+/** Has bad types. https://github.com/larryprice/novnc-core/issues/5 */
+import RFB from 'novnc-core';
+
+import { VncActions } from './VncActions';
+import constants from '../common/constants';
+
+import '@patternfly/react-styles/css/components/Consoles/VncConsole.css';
```

Comment:
You can write `import styles from '@patternfly/react-styles...';` and then later use `styles.consoleVnc` rather than the string `pf-c-console__vnc` if you'd like.